strawberry/utils/docstrings.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Optional
+
+import docstring_parser
+
+
+Docstring = docstring_parser.Docstring
+
+
+def get(obj: Any) -> Optional[Docstring]:
+    if obj is None or obj.__doc__ is None:
+        return None
+    return docstring_parser.parse(obj.__doc__)
+
+
+def type_description(docstring: Optional[Docstring]) -> Optional[str]:
+    if docstring is not None:
```

Comment:
can we do `if docstring is None: return None` so we don't nest the rest? 😊

Reply:
same below :)

Reply:
Done